packages/types/src/clerk.ts

```diff
@@ -472,6 +472,22 @@ export interface Resources {
 
 export type RoutingStrategy = 'path' | 'hash' | 'virtual';
 
+export type RoutingOptions =
```

Comment:
Let's not export this for now, as it is mostly used to DRY the other types internally. We can do it later if needed.

Ideally we wouldn't want to export `RoutingStrategy` either but since its a breaking change to do it now, let's leave it as is.